keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/kep.yaml

```diff
@@ -5,34 +5,38 @@ authors:
 owning-sig: sig-node
 participating-sigs: []
 status: implementable
-creation-date: "2024-05-08"
+creation-date: "2025-02-15"
 reviewers:
   - "@klueska"
   - "@ffromani"
 approvers:
   - "@sig-node-tech-leads"
-see-also: []
+see-also: 
+  - "keps/sig-node/2625-cpumanager-policies-thread-placement/"
+  - "keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/"
+  - "keps/sig-node/3545-improved-multi-numa-alignment/"
+  - "keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/"
+  - "keps/sig-node/4540-strict-cpu-reservation"
+  - "keps/sig-node/4800-cpumanager-split-uncorecache/"
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.31"
+latest-milestone: "v1.33"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   beta: "v1.31"
-  stable: "v1.32"
+  stable: "v1.33"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
   - name: "TopologyManagerPolicyBetaOptions"
-    components:
```

Comment:
why was this removed?

Reply:
I'm curious too, maybe vscode's markdown linter removed it, revert it now.